pandas/tests/io/parser/test_unsupported.py

```diff
@@ -140,10 +140,29 @@ def test_pyarrow_engine(self):
                 f"supported with the 'pyarrow' engine"
             )
             kwargs = {default: object()}
-            default_needs_bool = {"on_bad_lines", "error_bad_lines"}
+            default_needs_bool = {"warn_bad_lines", "error_bad_lines"}
             if default == "dialect":
                 kwargs[default] = "excel"  # test a random dialect
             elif default in default_needs_bool:
                 kwargs[default] = True
+            elif default == "on_bad_lines":
+                kwargs[default] = "warn"
             with pytest.raises(ValueError, match=msg):
                 read_csv(StringIO(data), engine="pyarrow", **kwargs)
+
+    @pytest.mark.parametrize(
+        "kwds",
+        [{"on_bad_lines": "warn"}, {"error_bad_lines": True}, {"warn_bad_lines": True}],
+    )
+    def test_pyarrow_bad_lines_fails(self, pyarrow_parser_only, kwds):
```

Comment:
Can this test be removed now? It seems to duplicate the ``test_pyarrow_engine`` test above now.

Reply:
good point, thx